mantis-control-plane/mantis-control-plane-server/src/test/java/io/mantisrx/master/api/akka/route/JacksonTest.java

```diff
@@ -67,16 +67,16 @@ public void testDeser4() throws IOException {
         mwm.setCluster(Optional.of("test"));
 
         final String out = objectMapper.writer(Jackson.DEFAULT_FILTER_PROVIDER).writeValueAsString(mwm);
-        assertTrue(out.contains("\"cluster\":{\"present\":true},"));
+        assertTrue(out.contains("\"cluster\":{\"empty\":false,\"present\":true}"));
```

Comment:
where and why is this "empty" key introduced?

Reply:
Had to do some digging into this.

This is a difference with JDK 8 vs JDK 11 (JDK i am using on M1: https://adoptium.net/download/). 
I thought setting the (inteliJ IDEA) File -> Project Structure- > Language level to 8 would be sufficient, however it dosent seem that way.
JDK 8 evaluates:
 `objectMapper.writeValueAsString(Optional.empty())` as `{"present":false}`
JDK 11 evaluates:
 `objectMapper.writeValueAsString(Optional.empty())` as `{empty":true,"present":false}`
 
 I will undo this change since the project compatibility is set Java 8